content/doc/book/installing/_docker-for-tutorials.adoc

```diff
@@ -11,73 +11,192 @@ the sibling file _docker.adoc (used in index.adoc).
+can be stopped later by running `docker stop jenkins-docker`.
+<4> Running Docker in Docker currently requires privileged access to function
+properly. This requirement may be relaxed with newer Linux kernel versions.
+// TODO: what versions of Linux?
+<5> This corresponds with the network created in the earlier step.
+<6> Makes the Docker in Docker container available as the hostname `docker`
+within the `jenkins` network.
+<7> Enables the use of TLS in the Docker server. Due to the use
+of a privileged container, this is recommended, though it requires the use of
+the shared volume described below. This environment variable controls the root
+directory where Docker TLS certificates are managed.
+<8> Maps the `/certs/client` directory inside the container to
+a Docker volume named `jenkins-docker-certs` as created above.
+<9> Maps the `/var/jenkins_home` directory inside the container to the Docker
+volume named `jenkins-data`. This will allow for other Docker
+containers controlled by this Docker container's Docker daemon to mount data
+from Jenkins.
+<10> ( _Optional_ ) Exposes the Docker daemon port on the host machine. This is
+useful for executing `docker` commands on the host machine to control this
+inner Docker daemon.
+<11> The `docker:dind` image itself. This image can be downloaded before running
+by using the command: `docker image pull docker:dind`.
++
+*Note:* If copying and pasting the command snippet above does not work, try
+copying and pasting this annotation-free version here:
++
+[source,bash]
+----
+docker run --name jenkins-docker --rm --detach \
+  --privileged --network jenkins --network-alias docker \
+  --env DOCKER_TLS_CERTDIR=/certs \
+  --volume jenkins-docker-certs:/certs/client \
+  --volume jenkins-data:/var/jenkins_home \
+  --publish 2376:2376 docker:dind
+----
 . Customise official Jenkins Docker image, by executing below two steps:
-.. Create Dockerfile with the folloing content:
+.. Create Dockerfile with the following content:
 +
 [source]
 ----
-FROM jenkins/jenkins:2.249.1-lts
+FROM jenkins/jenkins:2.249.2-slim
 USER root
-RUN apt-get update && apt-get install -y \
-       apt-transport-https \
-       ca-certificates \
-       curl \
-       gnupg2 \
+RUN apt-get update && apt-get install -y apt-transport-https \
+       ca-certificates curl gnupg2 \
        software-properties-common
 RUN curl -fsSL https://download.docker.com/linux/debian/gpg | apt-key add -
 RUN apt-key fingerprint 0EBFCD88
 RUN add-apt-repository \
        "deb [arch=amd64] https://download.docker.com/linux/debian \
-       $(lsb_release -cs) \
-       stable"
+       $(lsb_release -cs) stable"
 RUN apt-get update && apt-get install -y docker-ce-cli
 USER jenkins
-RUN /usr/local/bin/install-plugins.sh blueocean:1.24.0
-
-ENTRYPOINT ["/sbin/tini", "--", "/usr/local/bin/jenkins.sh"]
+RUN java -jar /usr/lib/jenkins-plugin-manager.jar --plugins blueocean:1.24.2
```

Comment:
should be `RUN jenkins-plugin-cli --plugins blueocean:1.24.2`

https://github.com/jenkinsci/docker#plugin-installation-manager-cli-preview-1

Reply:
Thanks for detecting that mistake!  Fixed in https://github.com/jenkins-infra/jenkins.io/pull/3922